src/modules/import/domain/import-event-handler.ts

```diff
@@ -64,9 +67,37 @@ export class ImportEventHandler {
             return this.logger.error(`No import data itemns found for Importvorgang:${importvorgangId}`);
         }
 
+        /* eslint-disable no-await-in-loop */
+        while (importDataItems.length > 0) {
+            const dataItemsToImport: ImportDataItem<true>[] = importDataItems.splice(
+                0,
+                this.NUMBER_OF_USERS_PRO_REQUEST,
+            );
+            await this.savePersonWithPersonenkontext(
+                importVorgang,
+                dataItemsToImport,
+                klassenByIDandName,
+                event.permissions,
+            );
+
+            importVorgang.incrementTotalImportDataItems(dataItemsToImport.length);
+            await this.importVorgangRepository.save(importVorgang);
+        }
+
+        importVorgang.finish();
+        await this.importVorgangRepository.save(importVorgang);
+    }
+
+    private async savePersonWithPersonenkontext(
+        importVorgang: ImportVorgang<true>,
+        dataItems: ImportDataItem<true>[],
+        klassenByIDandName: OrganisationByIdAndName[],
+        permissions: PersonPermissions,
+    ): Promise<void> {
+        const importDataItemsWithLoginInfo: ImportDataItem<true>[] = [];
         //We must create every peron individually otherwise it cannot assign the correct username when we have multiple users with the same name
         /* eslint-disable no-await-in-loop */
-        for (const importDataItem of importDataItems) {
+        for (const importDataItem of dataItems) {
```

Comment:
Wasn't added in this PR, but the point with the eslint-disable also applies here